src/content/caseStudies/codetv.mdx

```diff
@@ -0,0 +1,117 @@
+seo:
+  title: 'Unleashing creativity: How CodeTV built a video streaming platform with Astro and Mux'
+  description: 'How Astro gives CodeTV everything they need to run a subscription-based TV service.'
+coverGradientFrom: '#3245FF'
+coverGradientTo: '#BC52EE'
+publishDate: 'September 9, 2025'
+authors:
+  - astro-team
+lang: 'en'
+socialImage: /src/content/caseStudies/_images/codetv/social-image.webp
+coverImage: /src/content/caseStudies/_images/codetv/cover-image.webp
+headerImage: /src/content/caseStudies/_images/codetv/header.png
+---
+
+import showsGraphic from './_images/codetv/codetv-shows.webp';
+import coreWebVitalsGraphic from './_images/codetv/codetv-cwv.webp';
+import BlogContentImage from '~/components/BlogContentImage.astro';
+import ImageFigure from '~/components/ImageFigure.astro';
+import Blockquote from '~/components/Blockquote.astro';
+
+<div class="space-y-8 text-white">
+	<div class="text-2xl leading-normal tracking-normal text-astro-gray-100">
+		[CodeTV](https://codetv.dev/) features video programming for developers
+		ranging from live coding sessions to game shows. They needed a framework
+		that could handle a complex data architecture and deliver a high performance
+		streaming platform for subscribers.
+	</div>
+</div>
+
+Tired of every live-streamed event feeling the same, Jason Lengstorf set out to create engaging video content for developers.
+CodeTV's shows focus on original concepts and high production value to provide a true TV-like alternative for those tired of watching yet another Zoom call posted to YouTube.
+
+With Astro, Jason had the tools to build an entire media platform that handled content and performance needs and "just worked," keeping his primary focus on creating original programming.
+
+<ImageFigure caption="CodeTV’s offerings include multiple types of shows.">
+	<BlogContentImage
+		src={showsGraphic}
+		alt="Four TV screens featuring CodeTV shows: Web Dev Challenge, Leet Heat, Learn With Jason, and Unspoken Tech."
+	/>
+</ImageFigure>
+
+## Challenges
+
+When Jason Lengstorf found success with his web series "Learn with Jason," he realized that he wanted to produce more, and varied, video content. He dreamed of expanding his educational content to include scripted shows with characters you invest in, and game shows where devs face off head to head.
+
+But, to move in this new direction, he needed a new kind of website. Lengstorf realized, "I'm making TV. What does TV look like?" He noticed that established media streaming platforms such as Netflix, Hulu, and Nebula all followed a consistent architecture with specific features: intuitive navigation through several series, tracking watched content, notifications for new content, as well as self-serve subscriber controls for payments and account info.
+
+CodeTV needed to do a lot, and do it reliably, so that Lengstorf could spend his time creating content and not troubleshooting its delivery platform.
```

Comment:
```suggestion
But, to move in this new direction, he needed a new kind of website. Jason realized, "I'm making TV. What does TV look like?" He noticed that established media streaming platforms such as Netflix, Hulu, and Nebula all followed a consistent architecture with specific features: intuitive navigation through several series, tracking watched content, notifications for new content, as well as self-serve subscriber controls for payments and account info.

CodeTV needed to do a lot, and do it reliably, so that Jason could spend his time creating content and not troubleshooting its delivery platform.
```